src/views/Simulation/components/TopBar/MapEntities.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Checkbox,
+  MenuItem,
+  FormControl,
+  Select,
+  ListItemText,
+  InputLabel,
+  Chip,
+  Box,
+  Typography,
+  Switch,
+} from '@mui/material';
+import { useSimulationViewContext } from '../../SimulationViewContext';
+import { MAP_VIEW_FILTERS } from '../../constants/settings';
+
+export default function MapEntitiesDropdown() {
+  const { selectedMapFilters, setSelectedMapFilters, onlyCriticalPoints, setOnlyCriticalPoints } =
+    useSimulationViewContext();
+
+  const handleChange = (event) => {
+    const value = event.target.value;
+
+    if (value.includes('all')) {
+      setSelectedMapFilters(MAP_VIEW_FILTERS.map((e) => e.value));
+    } else {
+      setSelectedMapFilters(value);
+    }
+  };
+
+  const handleRemove = (value) => {
+    if (value === 'all') {
+      setSelectedMapFilters([]);
+    } else {
+      setSelectedMapFilters((prev) => prev.filter((v) => v !== value && v !== 'all'));
+    }
+  };
+
+  const renderValue = (selected) => {
+    if (selected.length === 0) return 'Entities';
+    if (selected.length === 1) {
+      const label = MAP_VIEW_FILTERS.find((opt) => opt.value === selected[0])?.label;
+      return label || 'Entities';
+    }
+    const firstLabel = MAP_VIEW_FILTERS.find((opt) => opt.value === selected[0])?.label;
+    const moreCount = selected.length - 1;
+
+    return (
+      <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+        <Chip
+          key={selected[0]}
+          label={firstLabel}
+          onDelete={() => handleRemove(selected[0])}
+          onMouseDown={(e) => e.stopPropagation()}
+          color="warning"
+          size="small"
+        />
+        <Chip label={`+${moreCount}`} size="small" sx={{ ml: 1 }} color="warning" />
+      </Box>
+    );
+  };
+
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center', gap: 2 }}>
+      <FormControl fullWidth variant="outlined" size="small">
+        <InputLabel id="map-entities-label">Entities</InputLabel>
```

Comment:
Not sure why, but the "_Entities_" placeholder is not vertically centered in the input field:

<img width="1153" height="78" alt="Image" src="https://github.com/user-attachments/assets/6dd1fddd-17f3-43c9-a266-7429ca3aa682" />